### ISSUE TITLE
[USER STORY] Alternative to running a SPARQL-endpoint

### ISSUE THREAD

**Issue by sskagemo** [STIRData/user-stories#40]
As a: Publisher of Business Register Data

I wish to: have a low cost / low maintenance alternative for publishing data in accordance with the STIRData specification

So that I: am able to publish Business Register Data in accordance with the STIRData specification, even though I don't have the necessary competence or resources to run a Sparql-endpoint

Related to Epic: #3 , #5 

Lowering the treshold for publishing data from the authoritiative source, might increase the adoption, and supports a transition from the current situation to the target state where.

**Comment by sskagemo** [comment]
Is this OK, @jakubklimek ?

**Comment by jakubklimek** [comment]
Yes, the solution is either "provide an RDF dump of the whole registry" - that is low cost, but does not allow the users to query the data. They need to spin up their own SPARQL endpoint to query the data or process it in another way. Linked Data Fragments in "half way" solution - it allows the users to query the data using only "triple pattern" queries - simple queries at the cost of increased network traffic - more data is returned than really necessary for the user. It is then up to the user to finish the query evaluation based on the given data, which is, on the other hand, smaller that the whole RDF dump. In addition, there is already a linked data fragments client implementation supporting most of SPARQL on top of a LDF server.

**Comment by sskagemo** [comment]
Would Linked Data Fragments open the possibility for doing a "traditional" REST API type of lookup, like for instance the current Open Data service of BRREG, like the following example that lookus up and returns data (subject to content negotiation) for the company with company-ID 815716272? Or would this be a require a third type of mechanism, in addition to SPARQL and Linked Data Fragments?
https://data.brreg.no/enhetsregisteret/oppslag/enheter/815716272

**Comment by jakubklimek** [comment]
This is possible both with SPARQL endpoints and Linked Data Fragments, as it involves developing a "REST API" wrapper on top of either one.
